google-cloud-spanner/lib/google/cloud/spanner/database.rb

```diff
@@ -422,13 +451,14 @@ def database_operations filter: nil, page_size: nil
         #     backup = job.backup
         #   end
         #
-        def create_backup backup_id, expire_time
+        def create_backup backup_id, expire_time, version_time = nil
```

Comment:
Here version_time is optional so better to use keyword argument.

`def def create_backup backup_id, expire_time, version_time: nil`

Reply:
Fixed.